tutorials/thermal_camera.md

```diff
@@ -0,0 +1,183 @@
+\page thermalcamera Thermal Camera
+
+In this tutorial, we will discuss how to use a thermal camera in Ignition Gazebo.
+
+There are currently a few limitations with the thermal camera, which will be mentioned at the end of the tutorial.
+
+## Setting up the thermal camera
+
+Here's an example of how to attach a thermal camera sensor to a model:
+
+```xml
+<model name="thermal_camera">
+  <pose>4.5 0 0.5 0.0 0.0 3.14</pose>
+  <link name="link">
+    <pose>0.05 0.05 0.05 0 0 0</pose>
+    <collision name="collision">
+      <geometry>
+        <box>
+          <size>0.1 0.1 0.1</size>
+        </box>
+      </geometry>
+    </collision>
+    <visual name="visual">
+      <geometry>
+        <box>
+          <size>0.1 0.1 0.1</size>
+        </box>
+      </geometry>
+    </visual>
+    <sensor name="thermal_camera" type="thermal">
+      <camera>
+        <horizontal_fov>1.047</horizontal_fov>
+        <image>
+          <width>320</width>
+          <height>240</height>
+        </image>
+        <clip>
+          <near>0.1</near>
+          <far>100</far>
+        </clip>
+      </camera>
+      <always_on>1</always_on>
+      <update_rate>30</update_rate>
+      <visualize>true</visualize>
+      <topic>thermal_camera</topic>
+    </sensor>
+  </link>
+  <static>true</static>
+</model>
+```
+
+Let's take a closer look at the portion of the code above that focuses on the thermal camera sensor:
+
+```xml
+<sensor name="thermal_camera" type="thermal">
+  <camera>
+    <horizontal_fov>1.047</horizontal_fov>
+    <image>
+      <width>320</width>
+      <height>240</height>
+    </image>
+    <clip>
+      <near>0.1</near>
+      <far>100</far>
+    </clip>
+  </camera>
+  <always_on>1</always_on>
+  <update_rate>30</update_rate>
+  <visualize>true</visualize>
+  <topic>thermal_camera</topic>
+</sensor>
+```
+
+As we can see, we define a sensor with the following SDF elements:
+* `<camera>`: The camera, which has the following child elements:
+    - `<horizontal_fov>`: The horizontal field of view, in radians.
+    - `<image>`: The image size, in pixels.
+    - `<clip>`: The near and far clip planes. Objects are only rendered if they're within these planes.
+* `<always_on>`: Whether the sensor will always be updated (indicated by `1`) or not (indicated by `0`).
+* `<update_rate>`: The sensor's update rate, in Hz.
+* `<visualize>`: Whether the sensor should be visualized in the GUI (indicated by `true`) or not (indicated by `false`).
```

Comment:
Should I add a note in, or would it make more sense to just remove `visualize` and `always_on` from the SDF file?

Reply:
let's make a note here for now. We just have not implemented these features yet. There are also talks about whether or not we should actually support the `<visualize>` param in in ign-gazebo since we now have gui plugins to toggle them on/off.

Reply:
I went ahead and made a note of this in https://github.com/ignitionrobotics/ign-gazebo/pull/442/commits/e2358592265682fc33fcb5706e8abf4b41525d13.